docs/src/reference-guide/components/graphql/pagination.md

```diff
@@ -0,0 +1,127 @@
+# Pagination
+
+The Fuel indexer currently supports offset-based pagination in order to allow users to selectively request parts of a set of results.
+
+A paginated query can be made using three keywords:
+
+- `order` - sort order (required)
+- `first` - limit on number of results (required)
+- `offset` - the amount of records to skip before returning results (optional)
+
+In order to generate a paginated response, your query should include a limit and either an ascending or descending sort order applied to one of the fields of the searched entity. _If you include a limit in your query, you are required to include a sort order as well._ Otherwise, your query will return an error.
```

Comment:
This is obviously correct, but I can't help but feel that with the lines 7,8 and 9, it explains all and any developer should understand the idea of `order`, `first` and `offset` params. It would read just as well (perhaps better?) without this paragraph altogether.